tests/src/test/scala/cats/tests/OptionTests.scala

```diff
@@ -17,6 +17,9 @@ class OptionTests extends CatsSuite {
   checkAll("Option[Int] with Option", TraverseTests[Option].traverse[Int, Int, Int, Int, Option, Option])
   checkAll("Traverse[Option]", SerializableTests.serializable(Traverse[Option]))
 
+  checkAll("Option with Unit", MonadErrorTests[Option, Unit].monadError[Int, Int, Int])
+  checkAll("MonadError[Option, Unit]", SerializableTests.serializable(MonadErrorTests[Option, Unit]))
```

Comment:
This is attempting to serialize `MonadErrorTests` instead of the `MonadError` instance for `Option`.